homework_3/homework3.py

```diff
@@ -24,5 +25,6 @@
 b=5
 c=6
 
-d=(2*5+6)**2
-print(d)
+d=(a*b+c)**a
```

Comment:
The value of `d` should be a result of concatenation variables `a, b, c` transformed to strings. Like this `d = str(a) + str(b) + str(c)`
Please, fix it